integration/qbft/scenarios/integration_test_type.go

```diff
@@ -157,8 +167,11 @@ func (it *IntegrationTest) Run() error {
 
 	var eg errgroup.Group
```

Comment:
using an errgroup here results that not all validators starts

Reply:
@nkryuchkov